src/sentry/static/sentry/app/components/fileChange.tsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {AvatarUser} from 'app/types';
+import AvatarList from 'app/components/avatar/avatarList';
+import {IconFile} from 'app/icons';
+
+type Props = {
+  filename: string;
+  authors: AvatarUser[];
+};
+
+function FileChange({filename, authors}: Props) {
+  return (
+    <FileItem className="list-group-item list-group-item-sm">
+      <Filename>
+        <StyledIconFile size="xs" />
+        {filename}
+      </Filename>
+      <div>
+        <AvatarList users={authors} avatarSize={25} typeMembers="authors" />
+      </div>
+    </FileItem>
+  );
+}
+
+FileChange.propTypes = {
+  filename: PropTypes.string.isRequired,
+  authors: PropTypes.array.isRequired,
+};
+
+const FileItem = styled('li')`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const Filename = styled('div')`
+  font-size: ${p => p.theme.fontSizeMedium};
+  ${overflowEllipsis}
+`;
+
+const StyledIconFile = styled(IconFile)`
+  color: ${p => p.theme.gray1};
```

Comment:
IconFile takes the color of the parent, but also accepts a `prop` color. Wouldn't it be better to pass the color directly?

Reply:
Passing the color as a prop would require importing the theme module. I've gotten mixed reviews on that in the past.